website/content/en/docs/building-operators/golang/quickstart.md

```diff
@@ -9,7 +9,7 @@ This guide walks through an example of building a simple memcached-operator usin
 
 ## Prerequisites
 
-- Go through the [installation guide][install-guide].
```

Comment:
Instead of removing `.` can you add one below

Reply:
Done